ext/node/polyfills/internal_binding/tcp_wrap.ts

```diff
@@ -141,6 +151,10 @@ export class TCP extends ConnectionWrap {
     }
   }
 
+  get fd() {
+    return this[fdSymbol];
```

Comment:
How about returning `this[kStreamBaseField]?.[internalFdSymbol]` here. That would return correct value even when `this[kStreamBaseField]` was changed after the constructor.

Reply:
It's a good shorthand way to return the `fd`. Is the following assumption true ? It looks like every time a client connects to a socket server, the `#accept` method creates a `new TCP(SOCKET, connection)` . And there seems to be no other way to mutate the value of the `fd`, other than the server accepting the connection to the client and setting the value of `fd`.
If it is true, then it is safe to set the value of fd in the TCP constructor, but regardless, your suggestion is neat way to return the `fd`. Thanks

Reply:
> it is safe to set the value of fd in the TCP constructor

I think that is true when TCP class is used by server, but it seems this class is also reused by client TCP connection. In that case, `[kStreamBaseField]` field is reassigned in `#connect` method